admin/data/sql/migrations/069.do.add-time-limits-to-group.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE mtc_admin.[group] ADD loadingTimeLimit tinyint NULL
```

Comment:
@GeorgeChatzigiannis both fields need to be decimal(5,2)

Reply:
good catch!